README.md

```diff
@@ -10,15 +10,35 @@ Clone this repository into your `themes` folder of your site. Issue the followin
 
 	git clone https://github.com/simonmika/hugo-theme-html5 themes/html5
 
-### Add Layout
-Add CSS to layout your site into:
+### Add stylesheets
+Add CSS for your site into the css folder. For example:
 
 	static/css/layout.css
+	static/css/colors.css
 
-### Add Color
-Add CSS to color your site into:
+And add it to your config under `.params.css`:
 
-	static/css/color.css
+#### TOML example:
+
```

Comment:
is that better? That's so weird, I was sure indented code blocks are valid Markdown, but those docs make no mention of it!